alicloud/resource_alicloud_ess_alarm.go

```diff
@@ -242,6 +271,7 @@ func resourceAliyunEssAlarmUpdate(d *schema.ResourceData, meta interface{}) erro
 		}
 	}
 	request.Dimension = &createAlarmDimensions
+	d.SetPartial("dimensions")
```

Comment:
`d.SetPartial` should be put after the API returned successfully.

Reply:
fixed